### PR TITLE
Enable setting cluster_id on KafkaRestMetricsContext

### PR DESCRIPTION
**Pull request by rnpridgeon**
This is not explicitly added to the config def as its not intended to actually be set manually by users at this time.

### CODE REVIEW

File: kafka-rest/src/main/java/io/confluent/kafkarest/KafkaRestConfig.java [hunk 1]
```diff
@@ -48,9 +48,10 @@
 
   private static final Logger log = LoggerFactory.getLogger(KafkaRestConfig.class);
   private final KafkaRestMetricsContext metricsContext;
-  private static final String TELEMETRY_PREFIX = "confluent.telemetry";
   private static final String METRIC_REPORTERS_PREFIX = "metric.reporters";
 
+  public static final String TELEMETRY_PREFIX = "confluent.telemetry";
```
Comment (rigelbm) [inline kafka-rest/src/main/java/io/confluent/kafkarest/KafkaRestConfig.java:53]: What is this prefix used for?

Comment (rigelbm) [reply kafka-rest/src/main/java/io/confluent/kafkarest/KafkaRestConfig.java:53]: I can see usages in test, but not in code.

File: kafka-rest/src/main/java/io/confluent/kafkarest/KafkaRestMetricsContext.java [hunk 2]
```diff
@@ -26,31 +26,24 @@
...
+  public static final String KAFKA_REST_RESOURCE_TYPE = "kafka_rest";
+  public static final String KAFKA_REST_RESOURCE_CLUSTER_ID_DEFAULT = "cluster_id_unavailable";
 
   public KafkaRestMetricsContext(String namespace, Map<String, Object> config) {
     metricsContext = new RestMetricsContext(namespace, config);
 
-    setResourceLabel(RESOURCE_LABEL_COMMIT_ID, AppInfoParser.getCommitId());
     setResourceLabel(RESOURCE_LABEL_TYPE, KAFKA_REST_RESOURCE_TYPE);
+    setResourceLabel(RESOURCE_LABEL_CLUSTER_ID,
+            (String) config.getOrDefault("bootstrap.cluster.id",
```
Comment (xvrl) [inline kafka-rest/src/main/java/io/confluent/kafkarest/KafkaRestMetricsContext.java:43]: I'm curious how this config gets defined? I couldn't find it in the codebase.

Comment (rnpridgeon) [reply kafka-rest/src/main/java/io/confluent/kafkarest/KafkaRestMetricsContext.java:43]: It's a follow-up PR coming to  ce-kafka-rest. I'll mark you as reviewer there as well. We  had to make a few changes with regard to how the  servlet was discovered/loaded registered with the embedded http server first but those have now been merged. I'll DM you the companion PR tomorrow morning.

Comment (rnpridgeon) [reply kafka-rest/src/main/java/io/confluent/kafkarest/KafkaRestMetricsContext.java]: https://github.com/confluentinc/ce-kafka-rest/pull/37

**Pull request merged**